pkg/rust/examples/hello_ast_builder.rs

```diff
@@ -0,0 +1,84 @@
+#[cfg(feature = "sled-storage")]
+mod hello_ast_builder {
+    use gluesql_core::ast_builder::Execute;
+    use {
+        gluesql::{
+            core::ast_builder,
```

Comment:
Sorry for the late review. Everything else looks nice :)
nit. could you merge this?
we can merge `Execute` into `gluesql::core::ast_builder`
```rust
use {
    gluesql::{
        core::ast_builder::{self, Execute},
    ..
```

Reply:
Thanks for reply! 

1. I update my fork up to the most recent commit of the origin repository.
2. I merge `Execute` into `gluesql::core::ast_builder`.

Please check when you can. :)